nifi-commons/nifi-record-path/src/main/java/org/apache/nifi/record/path/ArrayIndexFieldValue.java

```diff
@@ -59,6 +59,11 @@ public void updateValue(final Object newValue, final DataType dataType) {
         getParentRecord().get().setArrayValue(getField().getFieldName(), getArrayIndex(), newValue);
     }
 
+    @Override
```

Comment:
Would you explain why this method is empty in a nested comment and/or throw an UnsupportedOperationException?